test/controllers/articles_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ArticlesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
It's best to get some sort of test to assert your pages are rendering correctly.

Reply:
I'm going to get started on testing today!